frontend/src/concepts/pipelines/content/tables/columns.ts

```diff
@@ -124,6 +124,53 @@ export const pipelineRunColumns: SortableData<PipelineRunKFv2>[] = [
   kebabTableColumn(),
 ];
 
+export function getExperimentRunColumns(
+  metricsColumnNames: string[],
+): SortableData<PipelineRunKFv2>[] {
+  return [
+    { ...checkboxTableColumn(), isStickyColumn: true, stickyMinWidth: '45px' },
+    {
+      label: 'Run',
+      field: 'name',
+      sortable: true,
+      isStickyColumn: true,
+      hasRightBorder: true,
+      stickyMinWidth: '250px',
+      stickyLeftOffset: '45px',
```

Comment:
can/should these just use the number values like how the width param is (just a nit/question)

Reply:
These types are defined in Patternfly as strings for both the `Th` and `Td` table components.